infra/terraform/bootstrap/azure_devops.tf

```diff
@@ -13,7 +13,7 @@ resource "azuredevops_project" "project" {
   work_item_template = "Agile"
 
   lifecycle {
-    prevent_destroy = true # prevent destroying the project
+    # prevent_destroy = true # prevent destroying the project
```

Comment:
Is there a reason we are commenting this line out?

Reply:
No, this was from testing.  Good catch and updated!